starter-code/MyLinkedList.java

```diff
@@ -12,15 +12,45 @@ public int getSize() {
 	}
 
 	public boolean remove(int index) {
-		//to-do
+		if (index > size-1 || index < 0 || size == 0){
+			throw NullPointerException;
```

Comment:
If you throw an exception, then the return value is irrelevant, because any statements after the `throw` aren't executed. You should avoid throwing an exception manually unless you really need to. Here, it would be better to simply return `false` without throwing an exception.